internal/attestation/crafter/crafter.go

```diff
@@ -212,9 +184,32 @@ func LoadSchema(pathOrURI string) (*schemaapi.CraftingSchema, error) {
 		return nil, err
 	}
 
+	// Load, validate policies, and embed them in the schema
+	if err := validateAndPreloadPolicies(schema.GetPolicies().GetMaterials()); err != nil {
```

Comment:
Isn't this embedding the policy which we said we didn't want to do?

Reply:
I removed the embedding logic, but I should change the method name 😞 . Let me change it.